package.json

```diff
@@ -23,6 +23,9 @@
     "test:all": "jest --config=utils/test/jest.config.js",
     "version": "yarn run build && git add -A"
   },
+  "dependencies": {
+    "@testing-library/user-event": "12.1.0"
```

Comment:
I think this wants to be in `devDependencies`, right?

Reply:
Whoops, yeah that's right. I always forget to `-D` for `devDependencies`. I'll add a new commit to fix this.